Fronter/Source/Frames/MainFrame.cpp

```diff
@@ -99,6 +101,17 @@ void MainFrame::OnSupportUs(wxCommandEvent& event)
 	wxLaunchDefaultBrowser("https://www.patreon.com/ParadoxGameConverters");
 }
 
+void MainFrame::OnCheckForUpdates(wxCommandEvent& event)
+{
+	if (isUpdateAvailable(configuration->getConverterFolder() + "/configurables/version.txt", configuration->getConverterTagsGithubUrl()))
```

Comment:
does this work if there's no getConverterFolder()?
on linux I bet it'll break.

Reply:
Made the function break when `getConverterFolder()` returns an empty string.